examples/android/src/main/AndroidManifest.xml

```diff
@@ -2,10 +2,14 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.alohalytics.demoapp" >
 
-    <!-- This permission is needed if you want to access IMEI -->
-    <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
-    <!-- This one is needed to upload statistics to the server -->
+    <!-- REQUIRED to upload statistics to the server -->
```

Comment:
I'm appreciate a comment explaining what exactly do these `REQUIRED/`OPTIONAL` translate to.

Are they parsed by the build system? Are they notes to further developers, that they must / would be nice to have those permissions for the library to work?